fossology/uploads.py

```diff
@@ -36,10 +35,10 @@ def detail_upload(
         :Examples:
 
         >>> # Wait up to 20 minutes until the upload is ready
-        >>> long_upload = detail_upload(1, wait_time=120)
+        >>> long_upload = u.detail_upload(1, wait_time=120) # doctest: +SKIP
```

Comment:
```suggestion
        >>> long_upload = detail_upload(1, wait_time=120) # doctest: +SKIP
```

Reply:
fixed typo / leftover from test

BTW: I skipped (# doctest: +SKIP) some doctests in fossology/*.py. The current doctest environment has no "global scope"  saying each doctest block runs in it's own environment. To enable more doctests (remove those # doctest: +SKIP suffixes to a doctest line) some rework has to be done.